src/lib/primitives/jsonschema/__snapshots__/jsonschema.spec.ts.snap

```diff
@@ -1876,6 +1880,10 @@ Some useful links:
             "exportSymbol": false,
             "metaSchemas": {},
             "references": Map {
+              "#" => [
+                undefined,
```

Comment:
<i class='graphite__hidden'>[Re: lines 1884 to 1884]<br/></i>

just checking: is `undefined` expected here? the types say it should always be `name: string` in the first value of the tuple<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/skyleague/therefore/49?utm_source=unchanged-line-comment">Graphite</a>.</span>